charts/kafka-operator/README.md

```diff
@@ -53,7 +53,7 @@ The following table lists the configurable parameters of the Banzaicloud Kafka O
 Parameter | Description | Default
 --------- | ----------- | -------
 `operator.image.repository` | Operator container image repository | `ghcr.io/banzaicloud/kafka-operator`
-`operator.image.tag` | Operator container image tag | `v0.12.4`
+`operator.image.tag` | Operator container image tag | `v0.22.0-rc.1`
```

Comment:
**Question**: is that a good idea to update the readme with a release candidate tag?
I would suggest doing this only for release versions.

Reply:
yea I agree... but if we don't update readme, it is a mismatch with what the helm chart is deploying

Reply:
Oh, ok then.